### PR TITLE
updated file,

### PR DESCRIPTION
**Pull request by keviiixaviiii**
in my previous pull request #15663 

i updated file name, domain name also website is now accessible

### CODE REVIEW

File: domains/kevin-xavier.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "kevin-xavier.is-a.dev",
+  "owner": {
+    "email": "iamkevinxavi@gmail.com",
+    "github": "keviiixaviiii"
+  },
+  "records": {
+    "CNAME": "https://keviiixaviiii.github.io/My-Portfolio/"
```
Comment (DEV-DIBSTER) [inline domains/kevin-xavier.json:8]: Invalid CNAME, should just be a hostname without protocols and file paths.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by keviiixaviiii** [comment]
i updated the CNAME in json

File: domains/kevin-xavier.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "kevin-xavier.is-a.dev",
+  "owner": {
+    "email": "iamkevinxavi@gmail.com",
+    "github": "keviiixaviiii"
+  },
+  "records": {
+    "CNAME": "keviiixaviiii.github.io/My-Portfolio/"
```
Comment (Stef-00012) [inline domains/kevin-xavier.json:8]: CNAME can not contain `/`

**Review by Stef-DP — DISMISSED** [review]

File: domains/kevin-xavier.json (hunk 2 above)
Comment (keviiixaviiii) [reply domains/kevin-xavier.json]: Verify your is-a.dev Domain with GitHub Pages

Comment (keviiixaviiii) [reply domains/kevin-xavier.json]: created new Json file

Comment (keviiixaviiii) [reply domains/kevin-xavier.json]: recently updated file

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**